### PR TITLE
Issue #15 | Shopping List Screen

### LINKED ISSUE #15: 15. As a user, I want the shopping list screen to look pretty so the app looks professional and welcoming.
AC:

* The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

shopping-list

### LINKED ISSUE #16: 16. As a user, I want the item detail screen to look pretty so the app looks professional and welcoming.
AC:

* The item detail screen (the screen a user sees when they tap on one of their items in the shopping list) looks something like the following (or equivalent, use your creativity!):

item-detail

### PR DESCRIPTION
**Pull request by deboragaleano**
## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles:  Style   |
|  ✓ | 💡  New feature   |

## Acceptance Criteria

- [x]  The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

## Description

* **Styled the the list view**: changed the header icon, the navigation links and the flash messages
* **Extra:** Added a new `UserToken.js` component that allows the user to copy the list token to clipboard 

## Related Issue

closes #15 
Closes #16 

## Updates

Screen Shot 2020-11-27 at 6 30 43 PM

## Testing Steps / QA Criteria

* Pull down this branch with git pull origin `hn-dg-issue-15` and `git checkout hn-dg-issue-15`
* Run `yarn start` to launch the app to see the shopping list page 🌟
* Join a list by using the **tina swamp rung** token, or create your own list and add some items to it.

Now you should see how the list and navbar look after the final style.

**Thank you :)**

### CODE REVIEW

**Review by saraheisa — COMMENTED** [review]
Tres beau! Great work 🙌
Thanks for your effort and dedication to making the app look better✨

Only one thing I think you can also close #16 in this PR

File: src/components/AddItem.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const AddItem = () => {
             },
           )
           .then((success) => {
-            displayMessage('Successfully Added', 'success');
+            displayMessage('Your item was successfully added', 'success');
```
Comment (llunaplanet) [inline src/components/AddItem.js]: Cool, more clear message by giving more context 👍

**Review by llunaplanet — APPROVED** [review]
Good job! I love the new feature to show and copy the actual user token! 👏  LGTM

File: src/components/UserToken.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  };
+  return (
+    <div className="user-token">
+      <b>
+        List token: "<span id="user-token">{getToken()}</span>"
+      </b>
+      <button aria-label="Copy token" onClick={handleCopyToken}>
+        <i className="fa fa-clone"></i>
+      </button>
+    </div>
```
Comment (Lukabaramishvili) [inline src/components/UserToken.js:19]: I love this feature! Very clever of you 💡
One thing I could suggest is to add a 2-3 second `flash message` or just `text` indicating that the token was copied.

**Review by Lukabaramishvili — APPROVED** [review]
Wow, the app looks so nice! And it functions super nice. You can feel that it is well-architected! 👏 
Nothing blocking, just left a comment.

File: src/components/UserToken.js (hunk 2 above)
Comment (hajarNasr) [reply src/components/UserToken.js:19]: Thank you, Luka! Yes, that's actually a good idea. :)

**Pull request merged**